Test/Tests/RealtimeClientConnectionTests.swift

```diff
@@ -2402,55 +2402,49 @@ class RealtimeClientConnectionTests: XCTestCase {
     func test__063__Connection__connection_failures_once_CONNECTED__should_not_receive_published_messages_until_the_connection_reconnects_successfully() throws {
         let test = Test()
         let options = try AblyTests.commonAppSetup(for: test)
-        options.autoConnect = false
```

Comment:
I've picked up those changes from the previous PR, but shouldn't, `RTN15a` spec [wasn't changed](https://github.com/ably/specification/pull/88). So I've rolled back these changes - 003cbed96c0605398131080702f854d82b5e70f0

>I wonder if you understood the phrase "When a new connection is made with the correct connection recovery key" as meaning something other than what (I think) was intended?

No, I've failed to understand why recovery key was mentioned in this spec. Please share your thoughts on this.

Reply:
I'm skipping this test, because it's unclear why it was marked as `RTN15a` - 9172f99f6ca305f9a84906b648b817bc0edbdbb8
It also confuses me as "should not receive published messages until the connection reconnects successfully" is something that can't be otherwise since you can't receive anything via disconnected connection. I wonder what spec was really tested here if any. WDYT? @lawrence-forooghian

Reply:
Yeah, I don't know what was the intention of that test, either. Why did you choose to mark it as skipped instead of removing it, though?

Reply:
> No, I've failed to understand why recovery key was mentioned in this spec. Please share your thoughts on this.

I think it's confused — I think it should be talking about a [resume key](https://sdk.ably.com/builds/ably/specification/main/features/#RTN15b1), not a recovery key.

Reply:
Opened https://github.com/ably/specification/pull/170 to improve wording

Reply:
https://github.com/ably/ably-cocoa/pull/1714/commits/fc9b9bd2346cc551ddb5286943672b83d707b6ca